packages/client/src/features/tools/node-creation/node-creation-module.ts

```diff
@@ -30,7 +30,7 @@ import { InsertIndicatorView } from './node-creation-views';
 export const nodeCreationToolModule = new FeatureModule(
     (bind, unbind, isBound, rebind) => {
         const context = { bind, unbind, isBound, rebind };
-        bind(ContainerManager).toSelf().inSingletonScope();
+        bind(TYPES.IContainerManager).to(ContainerManager).inSingletonScope();
```

Comment:
```suggestion
        bindAsService(context, TYPES.IContainerManager, ContainerManager);
```

This ensures that the `ContainerManager` is also bound to itself and so for any adopters injecting it, the binding will still resolve (backwards compatibility).